tools/cbmc/patches/patch.py

```diff
@@ -3,24 +3,18 @@
 import subprocess
 import os
 import sys
+from glob import glob
 
 if os.path.isfile('patched'):
     sys.exit()
 
-patchfiles = [filename for filename in os.listdir('.')
-              if filename.endswith('.patch') ]
-
-tmpdir = 'c:\\temp\\' if sys.platform == 'win32' else '/tmp/'
-for patchfile in patchfiles:
-    tmpfile = tmpdir + patchfile
-    print("Patch file: {}".format(patchfile))
-    with open(patchfile) as inp, open(tmpfile, "w", newline="\n") as outp:
-        outp.write(inp.read())
-    cwd = os.getcwd()
-    result = subprocess.run(["git", "apply", tmpfile], cwd='../../..')
+for tmpfile in glob("*.patch"):
```

Comment:
Check that this code works under both bash and visual studio command prompt.

Reply:
I have run this code successfully in a 64-bit visual studio command prompt.